test/smoke/configured/configure_test.rb

```diff
@@ -0,0 +1,71 @@
+#
+# Cookbook:: nexus_repository_manager
+#
+# Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
+#
+
+# Inspec test for recipe nexus_repository_manager::configure
+
+describe file('/opt/sonatype/nexus/etc/fabric/hazelcast.xml') do
+  it { should exist }
+
+  context 'when multicast is enabled' do
```

Comment:
alternatively we could add something like that: https://gist.github.com/Fivell/8025849